src/tests/stdlib.rs

```diff
@@ -22,6 +22,22 @@ fn mkfile() -> (PathBuf, TempDir) {
     (file_path, temp_dir)
 }
 
+fn load_stdlib_test(func: &str) -> (String, String) {
+    let mut test = String::new();
+    fs::File::open(format!("./src/tests/stdlib/{func}.ab"))
+    .expect(&format!("Failed to open {func} test file"))
+    .read_to_string(&mut test)
+    .expect(&format!("Test {func} not found"));
+
+    let mut output = String::new();
+    fs::File::open(format!("./src/tests/stdlib/{func}.output.txt"))
```

Comment:
same goes here

also log files should be `.gitignore`d

Reply:
It isn't logging but just the output we need to match to check that the test works.

Reply:
yeah, but cmon it creates a file thats not useful to anyone but the machine it was runnning on. it is pretty much a log tho

Reply:
we need to put somewhere the output to match for the test itself and the idea is to not touch the `.rs` files.

Reply:
> we need to put somewhere the output to match for the test itself and the idea is to not touch the `.rs` files.

isnt it just like this?

`/src/tests/stdlib/*.output.txt`

Reply:
I think we misunderstood.
There is no log generated in a file and the txt mentioned is to match the tests so there isn't need to modify .gitignore